docs-js/faq.mdx

```diff
@@ -0,0 +1,27 @@
+---
+id: frequently-asked-questions
+title: Frequently Asked Questions
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: FAQ
+description: You asked! We answered! We collected here the most frequent question about SAP Cloud SDK for JavaScript.
+keywords:
+  - sap
+  - cloud
+  - sdk
+  - ai
+---
+
+### Which library should I use for AI integration in the SAP ecosystem? I see an npm package `CAP LLM Plugin`.
+
+Use the **SAP Cloud SDK for AI**, the only official Software Development Kit (SDK) for **SAP AI Core**, **SAP Generative AI Hub**, and **Orchestration Service**.
+The SAP Cloud SDK for AI is fully aligned with the SAP AI Core roadmap, ensuring you always have access to the latest features and the newest LLMs.
+
+The `CAP LLM Plugin` includes the following disclaimer:
+
+> This project is provided "as-is" with no expectation for major changes or support.
+
+As a result, new models (e.g., `gpt-4.1`, `gemini-2.0`, `anthropic--claude-3.7`, `mistralai`, and `deepseek`) and upcoming features of SAP AI Core will not be supported.
+
+Only use the plugin for special use cases or if you are an existing user.
+This recommendation is fully aligned with the CAP LLM Plugin team.
```

Comment:
Maybe add just a quick not that there are also Cloud SDK's for other languages as official tooling available?